### DISCUSSION TITLE
Bump version if another dependency changed

### DISCUSSION THREAD

**Discussion by max06** [renovatebot/renovate#29146] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

(I'm feeling kinda stupid at the moment... this is confusing me)

I have a repository with a directory containing helm-charts (v3). I managed to get updates working in `helm-values`, as well as using a RegexManager in the Chartfiles. To be able to release those, the Chart-versions need to be bumped as well. I'm aware this is a long story and that it's still not working by default.

Question: Can I instruct renovate to bump an artificial version string depending on another version change? To be precise I want renovate to use one of the two methods mentioned above to recognize a (for example) minor update, to subsequently bump the artificial version in the `Chart.yaml` by one minor version. 

Any help is appreciated!

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by kastl-ars** [comment]
Not sure if this is your usecase, but adding `"bumpVersion": "patch",` to the `renovate.json` in the repository leads to the chart version being updated if a dependency is changed.

```
---
apiVersion: v2
name: grafana-tempo-operator
version: 1.0.0
dependencies:
  - name: cert-manager
    repository: https://charts.jetstack.io
    version: 1.13.1
```

In the example above `1.0.0` is changed as `1.13.1` is noticed as outdated.

**Comment by kastl-ars** [comment]
And there is a working workaround including running a script in a postUpgradeAction to change things.

https://github.com/renovatebot/renovate/discussions/28861

**Comment by HonkingGoose** [comment]
I think this open issue is what you want?

- #4728

Somebody attempted to get it working, with this PR:

- #26441

But we had to revert that soon after, because it was broken (not sure how/why it was broken). The revert PR is:

- #26758

That's why the issue is still open.
